src/nsidc/metgen/config.py

```diff
@@ -30,6 +32,10 @@ class Config:
     checksum_type: str
     number: int
     dry_run: bool
+    filename_regex: Maybe[str] = Maybe.empty
+    time_coverage_duration: Maybe[str] = Maybe.empty
+    pixel_size: Maybe[str] = Maybe.empty
```

Comment:
@eigenbeam this line is not quite accurate -- I want to describe an optional integer, but the config file isn't read correctly (at least in the test environment) if I simply substitute `int` for `str`. Do you have references for a "maybe integer" implementation?